dockstore-webservice/src/main/resources/openapi3/openapi.yaml

```diff
@@ -4069,11 +3972,7 @@ paths:
           type: string
       responses:
         default:
-          content:
-            application/json: {}
           description: default response
-      security:
-      - bearer: []
```

Comment:
likely old version of yq

Reply:
What version do we need? @garyluu

Reply:
search for yq in the repo

Reply:
it's also in the primer

Reply:
No luck finding in primer.

Reply:
![image](https://user-images.githubusercontent.com/24548904/77086395-1dadc000-69d8-11ea-948a-ecefa8aa28ba.png)

Reply:
That is the old deprecated primer.